sql/src/test/java/io/crate/integrationtests/WhereSystemColumnIntegrationTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.integrationtests;
+
+import io.crate.action.sql.SQLActionException;
+import org.junit.Test;
+
+import java.util.Collections;
+
+import static org.hamcrest.Matchers.is;
+
+public class WhereSystemColumnIntegrationTest extends SQLTransportIntegrationTest {
+
+    @Test
+    public void testRangeQueryOnId() throws Exception {
+        execute("create table t1 (i int)");
+        execute("insert into t1 (i) values (1), (2), (3)");
+        refresh();
+        execute("select _id from t1 order by _id limit 1");
+        String _id = (String) response.rows()[0][0];
+
+        execute("select * from t1 where _id > ?", new Object[]{_id});
+        assertThat(response.rowCount(), is(2L));
+    }
+
+    @Test
+    public void testRangeQueryOnUid() throws Exception {
+        execute("create table t1 (i int)");
+        execute("insert into t1 (i) values (1), (2), (3)");
+        refresh();
+        execute("select _uid from t1 order by _uid limit 1");
+        String _uid = (String) response.rows()[0][0];
+
+        execute("select * from t1 where _uid > ?", new Object[]{_uid});
+        assertThat(response.rowCount(), is(2L));
+    }
+
+    @Test
+    public void testRangeQueryOnDocThrowsException() throws Exception {
```

Comment:
Would it make sense to do this as unit test instead in the LuceneQueryBuilderTest class? Maybe also the others, so that the tests aren't that expensive?